### PR TITLE
Cat-1949 comment out bricks

### PR DESCRIPTION
**Pull request by p4p4**
**reopens #1791**

### CODE REVIEW

File: catroid/src/main/res/layout/brick_loop_end.xml [hunk 1]
```diff
@@ -34,14 +34,14 @@
...
         style="@style/BrickContainer.Control.Small" >
 
         <TextView
             android:id="@+id/brick_loop_end_label"
             style="@style/BrickText.SingleLine"
             android:text="@string/brick_loop_end" />
-    </LinearLayout>
+    </org.catrobat.catroid.ui.BrickLayout>
 
 </LinearLayout>
```
Comment (sjaindl) [inline catroid/src/main/res/layout/brick_loop_end.xml]: add newline

File: catroid/src/main/res/layout/brick_loop_end_no_puzzle.xml [hunk 2]
```diff
@@ -44,6 +44,6 @@
             android:id="@+id/brick_loop_end_no_puzzle_label"
             style="@style/BrickText.SingleLine"
             android:text="@string/brick_loop_end" />
-    </LinearLayout>
+    </org.catrobat.catroid.ui.BrickLayout>
 
 </LinearLayout>
```
Comment (sjaindl) [inline catroid/src/main/res/layout/brick_loop_end_no_puzzle.xml]: add newline

File: catroid/src/main/res/layout/brick_loop_endless_no_puzzle.xml [hunk 3]
```diff
@@ -43,6 +43,6 @@
             android:id="@+id/brick_loop_endless_nopuzzle_label"
             style="@style/BrickText.SingleLine"
             android:text="@string/brick_loop_end" />
-    </LinearLayout>
+    </org.catrobat.catroid.ui.BrickLayout>
 
 </LinearLayout>
```
Comment (sjaindl) [inline catroid/src/main/res/layout/brick_loop_endless_no_puzzle.xml]: add newline

File: catroid/src/main/res/layout/brick_loop_endless.xml [hunk 4]
```diff
@@ -43,6 +43,6 @@
             android:id="@+id/brick_loop_endless_label"
             style="@style/BrickText.SingleLine"
             android:text="@string/brick_loop_end" />
-    </LinearLayout>
+    </org.catrobat.catroid.ui.BrickLayout>
 
 </LinearLayout>
```
Comment (sjaindl) [inline catroid/src/main/res/layout/brick_loop_endless.xml]: add newline

Comment (p4p4) [inline catroid/src/main/res/layout/brick_loop_endless.xml]: as in Robert C. Martin's ~Clean Code~: always leave a file in a better shape than you found it, right? :smiley:

**Pull request merged**